src/plugins/navigation_embeddable/public/components/navigation_embeddable_panel_editor.tsx

```diff
@@ -164,9 +184,12 @@ export const NavigationEmbeddablePanelEditor = ({
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiButton
-              disabled={!links || isEmpty(links)}
+              disabled={orderedLinks.length === 0}
```

Comment:
When editing an existing Links panel it is possible to remove all links. However, this leads to another empty state where you can't Save the changes. I'm not sure if this is wanted behavior or not. Presumably if the user is deleting the last link in a panel they should just want to delete the entire panel. But I also wonder if we should just allow users to have an empty panel? Maybe they will arrange the empty panel on the dashboard and add links later?

cc @andreadelrio @teresaalvarezsoler I'd love to get your thoughts, too.

Reply:
> I think we should make sure to add a tooltip to the disabled Save button explaining why it is disabled.

Good call - I think that's the right move :+1:

Reply:
How do we feel about this? 🤔 

https://github.com/elastic/kibana/assets/8698078/b9bee040-08e2-415b-ae0d-52395b8630c5

cc @gchaps for copy help here - not sure if `"Add at least one link to save."` is clear enough for the tooltip?

Reply:
The image embeddable also does not allow saving with an empty image no matter during creation or editing. So I think this makes sense. I also like the tooltip explaining why the Save button is disabled.

Reply:
I suggest simplifying the text

> Add one or more links

Users don't need a lot of nudging to understand what's going on here.

Reply:
Oooh yes, that is so much cleaner - thanks so much @gchaps 🙇

Reply:
Resolved by 928243c77d19b0dddcf294bbd7b40128d581a1ca

A quick a11y note about this - disabled buttons cannot receive focus, even if you manually set the tab index; so, this tooltip is inaccessible with keyboard controls. Seems like there's not much we can do about this until https://github.com/elastic/eui/issues/5666 is resolved.